jerry-core/lit/lit-magic-strings.inc.h

```diff
@@ -781,7 +781,10 @@ LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_SET_UTC_DATE_UL, "setUTCDate")
 #if JERRY_BUILTIN_STRING && JERRY_ESNEXT
 LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_STARTS_WITH, "startsWith")
 #endif
+#if JERRY_RESOURCE_NAME \
+|| !(JERRY_RESOURCE_NAME)
```

Comment:
I know this is auto-generated but `if (something || !something)` is totally meaningless.